build.savant

```diff
@@ -29,7 +29,7 @@ logbackVersion = "1.4.14"
 slf4jVersion = "2.0.13"
 testngVersion = "7.8.0"
 
-project(group: "org.primeframework", name: "prime-mvc", version: "4.22.12", licenses: ["ApacheV2_0"]) {
+project(group: "org.primeframework", name: "prime-mvc", version: "4.22.13", licenses: ["ApacheV2_0"]) {
```

Comment:
You're adding new features, so this cannot be a patch version. It would need to be `4.23.0`. 
Make sure also update the `pom.xml` and the `@Deprecated` annotation.

Reply:
Done.